x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.test.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render as reactRender, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { cloneDeep } from 'lodash';
+import {
+  BenchmarksSection,
+  TABLE_COLUMN_SCORE_TEST_ID,
+  TABLE_HEADER_TEST_ID,
+  TABLE_HEADER_SCORE_TEST_ID,
+} from './benchmarks_section';
+import { mockDashboardData, clusterMockData } from '../mock';
+import { TestProvider } from '../../../test/test_provider';
+import { KSPM_POLICY_TEMPLATE, CSPM_POLICY_TEMPLATE } from '../../../../common/constants';
+
+describe('<BenchmarksSection />', () => {
+  const render = (alterMockData = {}, dashboardType: 'kspm' | 'cspm' = KSPM_POLICY_TEMPLATE) =>
+    reactRender(
+      <TestProvider>
+        <BenchmarksSection
+          complianceData={{ ...mockDashboardData, ...alterMockData }}
+          dashboardType={dashboardType}
+        />
+      </TestProvider>
+    );
+
+  describe('Sorting', () => {
+    const mockDashboardDataCopy = cloneDeep(mockDashboardData);
```

Comment:
maybe a simpler approach would be to get the mock from a function and mutate new instances.